README.md

```diff
@@ -21,6 +21,13 @@ make
 
 See `make help` for help.
 
+### Generating test files
+
+Golden test files are re-generated via:
+```
+make ./... -update
```

Comment:
Can you change this to reflect the changes @TheoBrigitte is bringing here instead https://github.com/giantswarm/logging-operator/pull/451/files?

Reply:
```suggestion
make update-golden-files
```